src/main/java/com/joyent/manta/client/MantaClient.java

```diff
@@ -273,24 +226,24 @@ public MantaClient(final String url,
      * Deletes an object from Manta.
      *
      * @param path The fully qualified path of the Manta object.
-     * @throws IOException If an IO exception has occured.
-     * @throws MantaCryptoException If there's an exception while signing the request.
+     * @throws IOException If an IO exception has occurred.
+     * @throws com.joyent.manta.exception.MantaCryptoException If there's an exception while signing the request.
      * @throws MantaClientHttpResponseException If an HTTP status code {@literal > 300} is returned.
      */
-    public void delete(final String path) throws MantaCryptoException, MantaClientHttpResponseException, IOException {
-        LOG.debug(String.format("entering delete with path %s", path));
+    public void delete(final String path) throws IOException {
+        LOG.debug("DELETE {}", path);
+
         final GenericUrl genericUrl = new GenericUrl(this.url + formatPath(path));
         final HttpRequestFactory httpRequestFactory = httpRequestFactoryProvider.getRequestFactory();
         final HttpRequest request = httpRequestFactory.buildDeleteRequest(genericUrl);
-        // XXX: make an intercepter that sets these timeouts before each API call.
-        request.setReadTimeout(this.httpTimeout);
-        request.setConnectTimeout(this.httpTimeout);
 
         HttpResponse response = null;
         try {
             response = request.execute();
-            LOG.debug(String.format("got response code %s, header %s ",
-                                    response.getStatusCode(), response.getHeaders()));
+            if (LOG.isDebugEnabled()) {
```

Comment:
Oh, I remember why... We parameterize method calls (response.getStatusCode(), etc), so I have it ingrained in me as a habit to do a isXXEnabled() check around it when I am not doing simple interpolation with fields. Since, these are getters, it shouldn't matter.